FetchMigration/python/migration_monitor_counters.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class MigrationMonitorCounters:
+    prev_no_partitions_count: int = 0
+    prev_success_docs: int = -1
+    idle_success_doc_count: int = 0
+    seq_metric_api_fail: int = 0
+    seq_metric_value_fail: int = 0
```

Comment:
docs here would be great, especially why we care about each of them, what they mean.

Reply:
Sorry, this was bleed from a follow-up PR. This file has now been removed